app/.env.symlink

```diff
@@ -1 +1 @@
-../.env
\ No newline at end of file
+.env
```

Comment:
This is still not fixed. Make sure to do a self review before asking for review.